### PR TITLE
Improving hook_default_message_type documentation

### PR DESCRIPTION
**Pull request by wundo**
Added some optional values to the $values array to make sure other developers don't get lost.

### CODE REVIEW

**Comment by wundo** [comment]
This is related with #13

File: message.api.php [hunk 1]
```diff
@@ -73,7 +73,20 @@ function hook_message_view_alter(&$build) {
  */
 function hook_default_message_type() {
   $defaults['main'] = entity_create('message_type', array(
+    'description' => 'Type description',
+    'argument_keys' => array(
+      '!teaser',
+      '!body',
+      '@string'
```
Comment (RoySegall) [inline message.api.php]: Missing a comma.

File: message.api.php [hunk 2]
```diff
@@ -73,7 +73,20 @@ function hook_message_view_alter(&$build) {
...
+    ),
+    'message_text' => array(
+      LANGUAGE_NONE => array(
+        array('value' => 'Example text.'),
+      ),
+    ),
+    'language' => 'en'
```
Comment (RoySegall) [inline message.api.php]: Here as well.

**Comment by amitaibu** [comment]
Merged, thanks

**Comment by wundo** [comment]
Samething here: Closed with unmerged commits

**Pull request closed without merging**